<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LutConverterWidget</class>
 <widget class="QWidget" name="LutConverterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QToolButton" name="lutBrowseButton">
         <property name="text">
          <string>Browse...</string>
         </property>
         <property name="icon">
          <iconset resource="../duqf_resources.qrc">
           <normaloff>:/icons/set-preset</normaloff>:/icons/set-preset</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputEdit">
         <property name="text">
          <string/>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Input LUT file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Invert</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="invertBox">
       <property name="text">
        <string>Invert LUT</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Output format:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="outputFormatBox">
       <item>
        <property name="text">
         <string>Flame (*.3dl)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lustre (*.3dl)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cinespace (*.csp)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Houdini (*.lut)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Iridas (*.itx)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TrueLight (*.cub)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ICC (*.icc)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="iccOptionsWidget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ICC White point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="iccWhitePointBox">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>6505</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ICC profile target display</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="iccProfileEdit">
        <property name="placeholderText">
         <string>ICC profile</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="iccDescriptionEdit">
        <property name="placeholderText">
         <string>Short description</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Copyright</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="iccCopyrightEdit">
        <property name="placeholderText">
         <string>No copyright. Use freely.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="convertButton">
       <property name="text">
        <string>Convert LUT</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/ocio-bake-lut</normaloff>:/icons/ocio-bake-lut</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>304</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../duqf_resources.qrc"/>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
